### PR TITLE
[#ICC-322]  Replace danger with a github action

### PR DESCRIPTION
**Pull request by fabriziopapi**
#### List of Changes

see commit history

#### Motivation and Context

Danger is annoying to mantain due to token management.
We want to globally raplace it with a github action.

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/workflows/pr-title-linter-and-linker.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      - synchronize
+
+jobs:
+  lint:
+    name: Validate PR title And link Jira Issue
+    runs-on: ubuntu-22.04
+    steps:
+      - id: lint
+        # from https://github.com/pagopa/link-jira-issue-action
+        uses: pagopa/link-jira-issue-action@v1.0.1
```
Comment (pasqualedevita) [inline .github/workflows/pr-title-linter-and-linker.yaml:18]:
```suggestion
        # from https://github.com/pagopa/link-jira-issue-action/commits/main
        uses: pagopa/link-jira-issue-action@37d5e40820eddd388864905bee9c45ef50cc7453
```
Eng standards suggest to use commit hash

**Review by pasqualedevita — CHANGES_REQUESTED** [review]

File: .devops/code-review-pipelines.yml [hunk 2]
```diff
@@ -54,22 +48,6 @@ stages:
...
-        condition: 
-          and(
-            succeeded(),
-            ne(variables['DANGER_GITHUB_API_TOKEN'], 'skip')
-          )
-        steps:
-          - template: templates/node-job-setup/template.yaml@pagopaCommons
-
-          - bash: |
-              yarn danger ci
```
Comment (gquadrati) [inline .devops/code-review-pipelines.yml:67]: Remove danger from dependencies

Comment (Garma00) [reply .devops/code-review-pipelines.yml:67]: We can also remove `Dangerfile.ts` from `.eslintrc`

**Review by gquadrati — APPROVED** [review]
lgtm

**Pull request merged**